.github/workflows/ci.yml

```diff
@@ -199,7 +200,7 @@ jobs:
           echo "$(pwd)/.venv/bin" >> $GITHUB_PATH
 
       - name: Install extra dependencies
-        if: matrix.plugin == 'ravem' || matrix.plugin == 'zoom_rooms'
+        if: matrix.plugin == 'ravem' || matrix.plugin == 'zoom_rooms' || matrix.plugin == 'conversion'
```

Comment:
This shouldn't be needed, you don't use the zoom plugin nor `responses` in your plugin